server/models/PaymentMethod.js

```diff
@@ -14,16 +14,13 @@ import * as libpayments from '../lib/payments';
 import { isTestToken } from '../lib/stripe';
 
 import { maxInteger } from '../constants/math';
+import { services, types } from '../constants/paymentMethod';
```

Comment:
Maybe use a `*` import so you can later write `paymentMethod.services` and `paymentMethod.types` which is clearer.